### PR TITLE
Bug 1217234 - Fix tap on FM bookmark

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=db79a18696ebe17156bb92245164ba60327c3b98

File: apps/fm/js/fm.js [hunk 1]
```diff
@@ -670,6 +670,9 @@ var favoritesList = {
   },
 
   _getElemFreq: function(elem) {
+    // if we tap on the "Mhz" inner element
+    elem = elem.parentNode.classList.contains('fav-list-frequency') ?
+      elem.parentNode : elem;
```
Comment (justindarc) [inline apps/fm/js/fm.js]: Can't we simplify this whole thing with `elem.closest('.fav-list-item')` ?

Comment (hfiguiere) [inline apps/fm/js/fm.js]: we surely can. I actually forgot about ````closest````

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=033bd5fdb9d4f52fb02f0fc53fb890c921b3a75e

**Pull request merged**